src/TeachMeSkills/Homework/CreditCard.java

```diff
@@ -0,0 +1,29 @@
+package TeachMeSkills.Homework;
+import java.util.Scanner;
+
+public class CreditCard{
+    Scanner sc = new Scanner(System.in);
+    private double balance;
+    private final String accountNumber;
+    CreditCard(double balance, String accountNumber){
+        this.balance = balance;
+        this.accountNumber = accountNumber;
+    }
+    void incomingPayment(){
+        System.out.print("Введите сумму пополнения в белорусских рублях: ");
+        double sum = sc.nextDouble();
+        balance += sum;
+    }
+    void outgoingPayment(){
```

Comment:
This method should return remaining balance, or, e.g. -1< if we couldn't withdraw